docs/libcurl/opts/CURLINFO_ACTIVESOCKET.3

```diff
@@ -29,15 +29,16 @@ CURLINFO_ACTIVESOCKET \- get the active socket
 CURLcode curl_easy_getinfo(CURL *handle, CURLINFO_ACTIVESOCKET,
                            curl_socket_t *socket);
 .SH DESCRIPTION
-Pass a pointer to a curl_socket_t to receive the active socket used by this
-curl session. If the socket is no longer valid, \fICURL_SOCKET_BAD\fP is
-returned. When you finish working with the socket, you must call
-\fIcurl_easy_cleanup(3)\fP as usual on the easy handle and let libcurl close
-the socket and cleanup other resources associated with the handle. This is
-typically used in combination with \fICURLOPT_CONNECT_ONLY(3)\fP.
+Pass a pointer to a curl_socket_t to receive the most recently active socket
+used by this curl session. If the socket is no longer valid,
```

Comment:
Still seems unclear to me, how about _This option returns the active socket only after the transfer is complete, and is typically used in combination with \fICURLOPT_CONNECT_ONLY(3)\fP, which skips the transfer phase._

Reply:
In addition, it would be nice to know that it won't return a socket that's meant for asyn-ares resolving, but only a valid socket after connection is complete, for I use `CURLINFO_ACTIVESOCKET` w/ the multi socket interface to check when a connection is established.

Reply:
I've tried to accommodate both your comments in the updated commit. Have a look!